src/Aquifer.API/Endpoints/Resources/Content/MachineTranslation/Create/Endpoint.cs

```diff
@@ -34,7 +35,8 @@ public override async Task HandleAsync(Request req, CancellationToken ct)
             Content = req.Content,
             ContentIndex = req.ContentIndex,
             UserId = user.Id,
-            SourceId = req.SourceId
+            SourceId = req.SourceId,
+            RetranslationReason = req.RetranslationReason ?? null
```

Comment:
The default value for `RetranslationReason` is null, so I don't think we need the `?? null` here.